Traceback (most recent call last):
  File "/Users/kvarada/miniconda3/envs/jbook/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/kvarada/miniconda3/envs/jbook/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/kvarada/miniconda3/envs/jbook/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/Users/kvarada/miniconda3/envs/jbook/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/kvarada/miniconda3/envs/jbook/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/Users/kvarada/miniconda3/envs/jbook/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/kvarada/miniconda3/envs/jbook/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
cor = creditcard_df.corr()
plt.figure(figsize=(20, 10))
sns.set(font_scale=1)
sns.heatmap(cor, annot=True, cmap=plt.cm.Blues);
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[6], line 1[0m
[0;32m----> 1[0m cor [38;5;241m=[39m [43mcreditcard_df[49m[38;5;241;43m.[39;49m[43mcorr[49m[43m([49m[43m)[49m
[1;32m      2[0m plt[38;5;241m.[39mfigure(figsize[38;5;241m=[39m([38;5;241m20[39m, [38;5;241m10[39m))
[1;32m      3[0m sns[38;5;241m.[39mset(font_scale[38;5;241m=[39m[38;5;241m1[39m)

File [0;32m~/miniconda3/envs/cpsc330/lib/python3.10/site-packages/pandas/core/frame.py:10054[0m, in [0;36mDataFrame.corr[0;34m(self, method, min_periods, numeric_only)[0m
[1;32m  10052[0m cols [38;5;241m=[39m data[38;5;241m.[39mcolumns
[1;32m  10053[0m idx [38;5;241m=[39m cols[38;5;241m.[39mcopy()
[0;32m> 10054[0m mat [38;5;241m=[39m [43mdata[49m[38;5;241;43m.[39;49m[43mto_numpy[49m[43m([49m[43mdtype[49m[38;5;241;43m=[39;49m[38;5;28;43mfloat[39;49m[43m,[49m[43m [49m[43mna_value[49m[38;5;241;43m=[39;49m[43mnp[49m[38;5;241;43m.[39;49m[43mnan[49m[43m,[49m[43m [49m[43mcopy[49m[38;5;241;43m=[39;49m[38;5;28;43;01mFalse[39;49;00m[43m)[49m
[1;32m  10056[0m [38;5;28;01mif[39;00m method [38;5;241m==[39m [38;5;124m"[39m[38;5;124mpearson[39m[38;5;124m"[39m:
[1;32m  10057[0m     correl [38;5;241m=[39m libalgos[38;5;241m.[39mnancorr(mat, minp[38;5;241m=[39mmin_periods)

File [0;32m~/miniconda3/envs/cpsc330/lib/python3.10/site-packages/pandas/core/frame.py:1838[0m, in [0;36mDataFrame.to_numpy[0;34m(self, dtype, copy, na_value)[0m
[1;32m   1836[0m [38;5;28;01mif[39;00m dtype [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:
[1;32m   1837[0m     dtype [38;5;241m=[39m np[38;5;241m.[39mdtype(dtype)
[0;32m-> 1838[0m result [38;5;241m=[39m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_mgr[49m[38;5;241;43m.[39;49m[43mas_array[49m[43m([49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m[43m [49m[43mcopy[49m[38;5;241;43m=[39;49m[43mcopy[49m[43m,[49m[43m [49m[43mna_value[49m[38;5;241;43m=[39;49m[43mna_value[49m[43m)[49m
[1;32m   1839[0m [38;5;28;01mif[39;00m result[38;5;241m.[39mdtype [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m dtype:
[1;32m   1840[0m     result [38;5;241m=[39m np[38;5;241m.[39marray(result, dtype[38;5;241m=[39mdtype, copy[38;5;241m=[39m[38;5;28;01mFalse[39;00m)

File [0;32m~/miniconda3/envs/cpsc330/lib/python3.10/site-packages/pandas/core/internals/managers.py:1732[0m, in [0;36mBlockManager.as_array[0;34m(self, dtype, copy, na_value)[0m
[1;32m   1730[0m         arr[38;5;241m.[39mflags[38;5;241m.[39mwriteable [38;5;241m=[39m [38;5;28;01mFalse[39;00m
[1;32m   1731[0m [38;5;28;01melse[39;00m:
[0;32m-> 1732[0m     arr [38;5;241m=[39m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_interleave[49m[43m([49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m[43m [49m[43mna_value[49m[38;5;241;43m=[39;49m[43mna_value[49m[43m)[49m
[1;32m   1733[0m     [38;5;66;03m# The underlying data was copied within _interleave, so no need[39;00m
[1;32m   1734[0m     [38;5;66;03m# to further copy if copy=True or setting na_value[39;00m
[1;32m   1736[0m [38;5;28;01mif[39;00m na_value [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m lib[38;5;241m.[39mno_default:

File [0;32m~/miniconda3/envs/cpsc330/lib/python3.10/site-packages/pandas/core/internals/managers.py:1794[0m, in [0;36mBlockManager._interleave[0;34m(self, dtype, na_value)[0m
[1;32m   1792[0m     [38;5;28;01melse[39;00m:
[1;32m   1793[0m         arr [38;5;241m=[39m blk[38;5;241m.[39mget_values(dtype)
[0;32m-> 1794[0m     [43mresult[49m[43m[[49m[43mrl[49m[38;5;241;43m.[39;49m[43mindexer[49m[43m][49m [38;5;241m=[39m arr
[1;32m   1795[0m     itemmask[rl[38;5;241m.[39mindexer] [38;5;241m=[39m [38;5;241m1[39m
[1;32m   1797[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m itemmask[38;5;241m.[39mall():

[0;31mValueError[0m: could not convert string to float: 'C10001'

